src/Google.Cloud.Functions.Testing/FunctionTestServerBuilder.cs

```diff
@@ -72,8 +72,9 @@ public FunctionTestServerBuilder UseFunctionsStartups(IEnumerable<FunctionsStart
         }
 
         /// <summary>
-        /// Examines <paramref name="attributedType"/> (and its base class hierarchy) for <see cref="FunctionTestStartupAttribute" />
-        /// annotations. If any such attributes are found, the startup classes specified by the attributes are used
+        /// Examines <paramref name="attributedType"/> (and its base class hierarchy, and the assembly in which it's declared)
+        /// for <see cref="FunctionsStartupAttribute" /> annotations.
+        /// If any such attributes are found, the startup classes specified by the attributes are used
         /// instead of the startup classes specified in the assembly containing the function target. If no attributes
```

Comment:
"instead of the usual startup classes for the target function..."

Reply:
Done.